espnet2/speechlm/core_lm/valle.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python3
```

Comment:
Can you elaborate a bit on the relationship between ar_nar and valle? I'm not quite sure about how their design is formulated.

Reply:
ar_nar is deprecated.